proposals/PROPOSALS.md

```diff
@@ -0,0 +1,156 @@
+
+1. **Accepted**, in which case the HF Technical Track has accepted the proposal, and it will be merged by the HF Technical Track members.
+2. **Rejected**, in which case the HIP is closed and no longer evaluated in the future.
+3. **Postponed**, in which case the HF Technical Track sets aside the HIP under some conditions. When those conditions are met, the HIP can be resubmitted.
+4. **Under revision**, in which case the author needs to continue the formal evaluation and address all the HF Technical Track feedback. Thus, the follow-up discussion is scheduled for the next iteration. 
+5. **Dormant**,  in which case no changes have been made to a HIP in two iterations, it’s marked as dormant and both the PR and issue are closed. Dormant HIPs can be reopened by any person, be it the same or different authors, at which point it will start from the formal evaluation phase.
+
+#### Involving Stakeholders
+
+At this point where a propsal is being reviewed, if there are relevant stakeholders from industry or in the community (e.g. library authors and maintainers who are affected), they should be solicited for comment and brought into the discussion. Remember, proposals may sound good, but it's best to iron out as many details as possible prior to their acceptance. This means that any hesitations stakeholders have with the project should be factored into the proposal's details. 
+
+#### Updating Proposal Statuses
+
+If the author of a particular proposal wants to update the status of their proposal, they should either comment on the issue and tag an HF Technical Track member with a note regarding what status they would like to update the proposal to. It is good to be redundant and raise the question in our other fora as well: Slack and Discourse. However, Github should be the first choice. 
+
+### Formal presentation 
+
+After a proposal is merged, the author will present KPI's and deliverables that define the proposal to the HF Technical track. During the next available Technical Track meeting after acceptance, the author or designated lead (as defined by the proposal) should give a short 5-10min presentation consisting of the following: 
+
+1. **High level summary:** the author will present the high-level abstract for what the proposal intends to accomplish. 
+2. **Timeline:** In what timeline, and should outline possible hurdles and risks that affect the timeline. 
+3. **Resources:** What resources are required in order to take the project from start to completion.
+4.  **Contact:** Who is going to update the HF Technical Track members, and on what schedule.
+
+Once the formal presentation is given, the HF Technical Track members will track the progress over time at our ongoing meetings. This also gives the track members time to introduce themselves and and we can all get to know each other. Currently, HF Technical Track meetings are held weekly on Tuesdays at 11:30am-12:30pm UTC-5 (EST). 
+
+## Structure of the process
+
```

Comment:
This is a bit of an odd section.  What would be lost if we deleted it?

Reply:
I wouldn't want to scrap it completely, because it tells precisely how voting works, who will be voting, and so on. I'm fine with losing the preamble and the first section (The HIP authors). Those don't bring much.

Reply:
Agreed - removed in latest commits